src/scss/_type-mixins.scss

```diff
@@ -0,0 +1,102 @@
+/// Outputs font-family, size and line-height, and progressive
+/// font loading styles for Serif font
+///
+/// @param {Bool | Number} $scale [false] - number of the scale to use
+/// @param {Bool | Number} $line-height [false] - multiple of the baseline unit for bottom margin
+/// @param {Bool} $progressive [true] - multiple of the baseline unit for bottom margin
+@mixin oTypographySerif($scale: false, $line-height: false, $progressive: false) {
+	font-family: $o-typography-serif;
+
+	@if $scale {
+		@include oTypographySize($scale, $line-height);
+	}
+
+	// AO: This is not a progressive font so won't output anything
+	// is it worth keeping this anyway?
```

Comment:
I think just change this comment to say that progressive will probaly always be false for serif unless our serif font stops being Georgia